src/main/java/org/opentripplanner/middleware/utils/GraphQLUtils.java

```diff
@@ -0,0 +1,51 @@
+package org.opentripplanner.middleware.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+
+public class GraphQLUtils {
+    private static final Logger LOG = LoggerFactory.getLogger(GraphQLUtils.class);
+
+    // Lazily-initialized in getPlanQueryTemplate()
+    private static String planQueryTemplate = null;
+
+    /**
+     * Location of the GraphQL plan query template file, as Java resource.
+     */
+    public static final String PLAN_QUERY_RESOURCE = "queries/planQuery.graphql";
+
+    /**
+     * Return the full GraphQL plan file planQueryTemplate in Java string format, with {@code "} as {@code \"}
+     */
+    public static String getPlanQueryTemplate() {
+        if (GraphQLUtils.planQueryTemplate == null) {
+            GraphQLUtils.planQueryTemplate = planQueryTemplateAsString(PLAN_QUERY_RESOURCE);
+	}
+        return GraphQLUtils.planQueryTemplate;
+    }
+
+    /**
+     * Return a GraphQL planQueryTemplate in Java string format, with {@code "} as {@code \"}
+     * @param resource the plan file or any GraphQL file
+     */
+    static String planQueryTemplateAsString(String resource) {
```

Comment:
SonarLint: Remove the resource method parameter and refer to `GraphQLUtils.PLAN_QUERY_RESOURCE` directly.